save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityView.kt

```diff
@@ -50,7 +50,7 @@ import kotlinx.serialization.json.Json
     "TYPE_ALIAS",
     "EMPTY_BLOCK_STRUCTURE_ERROR",
 )
-val fossGraph: FC<FossGraphViewProps> = FC { props ->
+val vulnerability: FC<FossGraphViewProps> = FC { props ->
```

Comment:
```suggestion
val vulnerabilityView: FC<FossGraphViewProps> = FC { props ->
```

And also need to change `FossGraphViewProps -> VulnerabilityViewProps`

Reply:
done, also renamed boss package